5.5/run.sh

```diff
@@ -6,7 +6,7 @@ if [ "${1:0:1}" != '-' ]; then
   exec "$@"
 fi
 
-exec java -jar lib/sonar-application-$SONAR_VERSION.jar \
+exec java $SONARQUBE_JAVA_OPTS -jar lib/sonar-application-$SONAR_VERSION.jar \
```

Comment:
@cmroche note that `sonar-application.jar` just starts and monitors other java processes (such as web server, elastic search), so setting `-Xmx` has no effect on them. Settings for those processes can be placed into `conf/sonar.properties` (see [`sonar.web.javaOpts` for example](https://github.com/SonarSource/sonarqube/blob/5.5/sonar-application/src/main/assembly/conf/sonar.properties#L93)). Or specified as parameters to `sonar-application.jar` and so already supported by this image without need to introduce any additional environment variables:
```
docker run -d --name sonarqube -p 9000:9000 sonarqube -Dsonar.web.javaOpts=-Xmx512m
```